### PR TITLE
PLT-3268 Made keyboard shortcuts open/close account settings and channel switcher

### PR DESCRIPTION
**Pull request by DavidLu1997**
#### Summary
CTRL/CMD+SHIFT+A now opens/closes the account settings modal
CTRL/CMD+K now opens/closes the channel switcher modal

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3268

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] Has driver changes that have been merged and package.json updated
- [ ] Has enterprise changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization files updated
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

File: webapp/components/navbar.jsx [hunk 1]
```diff
@@ -180,9 +180,11 @@ export default class Navbar extends React.Component {
...
         if (Utils.cmdOrCtrlPressed(e) && e.keyCode === Constants.KeyCodes.K) {
             e.preventDefault();
-            this.setState({
-                showChannelSwitchModal: true
-            });
+            if (this.state.showChannelSwitchModal === false) {
+                this.setState({showChannelSwitchModal: true});
+            } else {
+                this.setState({showChannelSwitchModal: false});
+            }
```
Comment (jwilander) [inline webapp/components/navbar.jsx]: Wouldn't it be simpler to just do `this.setState({showChannelSwitchModal: !this.state.showChannelSwitchModal});` if it's just toggling each time?

File: webapp/components/navbar_dropdown.jsx [hunk 2]
```diff
@@ -77,7 +77,11 @@ export default class NavbarDropdown extends React.Component {
     openAccountSettings(e) {
         if (Utils.cmdOrCtrlPressed(e) && e.shiftKey && e.keyCode === Constants.KeyCodes.A) {
             e.preventDefault();
-            this.setState({showUserSettingsModal: true});
+            if (this.state.showUserSettingsModal === false) {
+                this.setState({showUserSettingsModal: true});
+            } else {
+                this.setState({showUserSettingsModal: false});
+            }
```
Comment (jwilander) [inline webapp/components/navbar_dropdown.jsx]: Same for here

**Comment by DavidLu1997** [comment]
@jwilander changes made

**Comment by jwilander** [comment]
+1

**Pull request merged**